components/LocalePicker.php

```diff
@@ -142,7 +150,29 @@ protected function makeLocaleUrlFromPage($locale)
             $router = new RainRouter;
 
             $params = $this->getRouter()->getParameters();
-
+                        
+            /**
+             * @event translate.params.translateParams
+             * Enables manipulating the URL parameters
+             *
+             * You will have access to the page object, the old and new locale and the URL parameters.
+             *
+             * Example usage:
+             *
+             * Event::listen('translate.params.translateParams', function($page, $params, $oldLocale, $newLocale) {
```

Comment:
Please add four spaces of indentation for the example code in this comment. When we get API documentation generation for plugins, this will properly format the example as a code block in the documentation